rules/xcodeproj.bzl

```diff
@@ -342,12 +348,14 @@ xcodeproj = rule(
         "_xcodeproj_installer_template": attr.label(executable = False, default = Label("//tools/xcodeproj_shims:xcodeproj-installer.sh"), allow_single_file = ["sh"]),
         "_infoplist_stub": attr.label(executable = False, default = Label("//rules/test_host_app:Info.plist"), allow_single_file = ["plist"]),
         "_workspace_xcsettings": attr.label(executable = False, default = Label("//tools/xcodeproj_shims:WorkspaceSettings.xcsettings"), allow_single_file = ["xcsettings"]),
+        "output_processor": attr.label(executable = True, default = Label("//tools/xcodeproj_shims:output-processor.rb"), cfg = "host", allow_single_file = ["rb"]),
```

Comment:
if we're going to allow people to override the default for this attribute (which is what's happening here, since the name isn't `_` prefixed), there's no need to limit to only `.rb` files

Reply:
Going with the `allow_single_file = True` approach here instead